lib/logstash/outputs/dynatrace.rb

```diff
@@ -112,10 +111,6 @@ def multi_receive(events)
 
         @logger.debug("Successfully exported #{events.length} events with #{retries} retries")
       end
-
-      def send(request)
```

Comment:
Instead of mocking my own send method I'm now mocking it on the Net::HTTP client